model.md

```diff
@@ -0,0 +1,6 @@
+# Model
+
+The *verification* problem can be solved by cryptographic signing (provided by IPFS)  
+The *authority* problem can be solved by webs of trust (provided by Data Together)  
+The *discovery* problem can be solved by ??? (provided by Data Together)  
+The *access* problem feels like a complex one, actually.
```

Comment:
New text from October 26: _The decentralized (peer-to-peer) web allows [you | groups] to possess data you rely on and address information by _what_ it is rather than _where_ it is._